samples/bluetooth/broadcast_audio_sink/boards/nrf52840dk_nrf52840.conf

```diff
@@ -4,3 +4,5 @@ CONFIG_BT_CTLR_SYNC_ISO=y
 
 # Supports the highest SDU size required by any BAP LC3 presets (155)
 CONFIG_BT_CTLR_SYNC_ISO_PDU_LEN_MAX=155
+# Supports the highest advertising data that can be set in a single HCI command
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=251
```

Comment:
Should it be that 252 bytes be supported?

Reply:
Chose 251 specifically as I set `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=251` because that is largest extended advertising value for a single HCI command (and surely is enough for this sample, and I think it makes more sense if the two values in the pair of samples are equal. 

I did consider 252 to support the maximum single periodic advertising report, but felt that the 251 would be better to cover both extended and periodic advertising